docs/tempo/website/configuration/_index.md

```diff
@@ -717,16 +717,20 @@ overrides:
     
     # Global ingestion limits configurations
 
+    # Specifies whether the ingestion rate limits should be applied by each replica (of the distributor and ingester)
+    # individually OR should be shared across all replicas. See the "override strategies" section below for an example.
```

Comment:
Wording suggestions here are to attempt to use similar terms across all documentation.  Here, we're talking about components, so I've substituted "instance" for "replica." 

```suggestion
    # Specifies whether the ingestion rate limits should be applied by each instance of the distributor and ingester
    # individually, or the limits are to be shared across all instances. See the "override strategies" section for an example.
```